src/platform_impl/windows/window.rs

```diff
@@ -1189,8 +1189,8 @@ where
     Ok(initdata.window.unwrap())
 }
 
-unsafe fn register_window_class<T: 'static>() -> Vec<u16> {
-    let class_name = util::encode_wide("Window Class");
+unsafe fn register_window_class<T: 'static>(window_classname: &String) -> Vec<u16> {
```

Comment:
With the change suggested above we can simplify it a bit to
```suggestion
unsafe fn register_window_class<T: 'static>(class_name: &[u16]) {
```